tests/src/test/java/alluxio/rest/RestApiTest.java

```diff
@@ -34,6 +34,7 @@
       .setProperty(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false")
       .setProperty(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL.getAuthName())
       .setProperty(PropertyKey.MASTER_STARTUP_CONSISTENCY_CHECK_ENABLED, "true")
+      .setProperty(PropertyKey.USER_LINEAGE_ENABLED, "true")
```

Comment:
ideally not, but the `@Config(confParams = {PropertyKey.Name.USER_LINEAGE_ENABLED, "true"})` was not taking effect ... do you propose any alternative solution?